apps/supernova/README.md

```diff
@@ -23,7 +29,7 @@ Alerting rules offer the capability to define alert conditions using expressions
 
 #### Labels
 
-The labels clause allows specifying a set of additional labels to be attached to the alert. Following is a live example of a set of labels from an alert of the `support group:containers` with `severity:info` in the `region:eu-de-2`.
+The labels clause allows specifying a set of additional labels to be attached to the alert. Following is a live example of a set of labels from an alert of the `support_group:containers` with `severity:info` in the `region:eu-de-2`.
```

Comment:
Do we want to still mention the `support_group:containers` from the live example?